src/main/java/domain/pages/DirectoryPage.java

```diff
@@ -53,19 +60,39 @@ public void clickListByStateandDistrict(){
         return representativeNames;
     }
 
-    public String getPartyXpath(String xpathParty, String representativeName){
-        return driver.findElement(By.xpath(String.format(xpathParty, representativeName))).getText();
+    public String getParty(String xpathParty, String representativeName){
+        return driver.findElement(By.xpath(String.format(xpathParty, representativeName))).getText().trim();
     }
 
-    public char getPartyByRepresentative(String representativeName){
-        return getPartyXpath(partyXpathByName, representativeName).charAt(0);
+    public String getPartyByRepresentative(String representativeName){
+        return getParty(partyXpathByName, representativeName);
     }
 
-    public List<String> getAllStates() {
+    public List<String> getAllStatesSorted() {
         List<String> allStatesArray = new ArrayList<String>();
         for (WebElement allState:allStates){
             allStatesArray.add(allState.getText());
         }
-        return allStatesArray;
+        //List <String> startingWithAStates = allStatesArray.stream().filter(s->s.startsWith("A")).collect(Collectors.toList());
+        //return startingWithAStates;
+        List <String> sortedStates = allStatesArray.stream().sorted().collect(Collectors.toList());
+        return sortedStates;
+    }
+
+    public List<WebElement> allRowInfoForSelectedState(String state){
+        List<WebElement> rows = driver.findElements(By.xpath(String.format(rowInfoXpath, state)));
+        return rows;
+    }
+
+    public boolean checkFormatInTheRowsForSelectedState(String state){
```

Comment:
rename method isFormatInTheRowsForSelectedStateCorrect

Reply:
nobody checks the format by the way

Reply:
done

Reply:
thanks for the clarification